session4/Assignment1/Assignment1.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+import java.util.List;
+
+class Assignment1

```

Comment:
Do not give class name like Assignment1 or 2 or 3 like this. Give meaning name to the class, .java file and methods. For example you can use class name like: SolvePermutaion, SolveTowerOfHonnai, NQueenProblem.
And give java file name like this: SolvePermutaion.java

Note: -Naming is most important thing in project work becuase multiple developers work on same code, so other developers should understand what that class contains just by reading its name only.